pkgs/development/compilers/hhvm/default.nix

```diff
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
   name    = "hhvm-${version}";
-  version = "3.21";
+  version = "3.23";
```

Comment:
Could you declare the actual version, `3.23.2`, and use `rev = "HHVM-${version}"` below?